presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -112,6 +116,22 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
                 new String[] {"TABLE", "VIEW", "MATERIALIZED VIEW", "FOREIGN TABLE"});
     }
 
+    @Override
+    public PreparedStatement buildSql(ConnectorSession session, Connection connection, JdbcSplit split, List<JdbcColumnHandle> columnHandles)
+            throws SQLException
+    {
+        return new QueryBuilder(identifierQuote, (columnHandle) -> "?::" + columnHandle.getJdbcTypeHandle().getJdbcTypeName()).buildSql(
```

Comment:
please extract `(columnHandle) -> "?::" + columnHandle.getJdbcTypeHandle().getJdbcTypeName()` as method, then replace it with `this::yourNewMethodName`